src/WebView.vala

```diff
@@ -160,6 +160,21 @@ public class Mail.WebView : WebKit.WebView {
         }
     }
 
+    public new void load_plain_text (string plain_text) {
```

Comment:
Please do not use the `new` keyboard to hide the parent class method but either:
 * create a new method with a different name
 * if `load_plain_text` it's only used once, replace it with the content of the code.

Explanation:
the new keyboard allow you to use the same method name as one of the parent ones but it has some hidden consequences in the code:
If you have the webview somewhere in the code declared as `Webkit.WebView` and that you call `load_plain_text` then it won't call this method but the one in `Webkit.WebView`
I think that the behavior you want is the one used with the `override` keyboard when it would actually replace the original function for all calls. But as this method is not `virtual` not `abstract` you can't use the `override` keyword.

Reply:
Hmmm, do you think we should rename `new void load_html (...)` to `void set_html (...)` too them? I thought reusing the `load_plain_text` name would be more consistent with `load_html`.

Since `load_plain_text` is only called once, I've inlined the code there (in MessageListItem). -> One more thing that can move into a utility class later :)